.github/workflows/ros1.yml

```diff
@@ -56,6 +56,37 @@ jobs:
         ros-distro: ${{ matrix.distro }}
         gazebo-version: ${{ matrix.gazebo }}
         workspace-dir: simulation_ws
+     - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_ROS1 }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_ROS1 }}
+        aws-region: ${{ secrets.AWS_REGION }}
+      if: github.event_name == 'schedule' && github.ref == 'refs/heads/ros1'
```

Comment:
do we have a guidelines which quote to use for string (single ' vs double ") ?

Reply:
[Github actions manual](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#literals) sugggests single quotes for strings